public/app/plugins/panel/canvas/CanvasContextMenu.tsx

```diff
@@ -215,6 +215,11 @@ export const CanvasContextMenu = ({ scene, panel }: Props) => {
         currentLayer.doAction(actionType, currentSelectedElement);
```

Comment:
Do we also want to maintain selection on send to back / front cases?

Reply:
Yeah that's a good point - I think this is great as is, just wanted to raise that question :)